cli/src/cmd/commands/commands.go

```diff
@@ -0,0 +1,19 @@
+package commands
+
+import (
+	"cli/cmd/config"
+	"cli/cmd/pkg"
+	"cli/cmd/project"
+	"cli/cmd/stack"
+
+	"github.com/spf13/cobra"
+)
+
+func AddCommands(cmd *cobra.Command) {
+	cmd.AddCommand(
+		config.DeclareConfigCommand(),
+		pkg.DeclarePackageCommand(),
+		project.DeclareProjectCommand(),
+		stack.DeclareStackCommand(),
```

Comment:
This would be a good place to remove stack, I think the others do something but are incomplete, also I dont think stack needs to be in here anymore as the profile feature covers this use case

Reply:
I've removed all non-complete options in [7f85219](https://github.com/openhie/package-starter-kit/pull/19/commits/7f85219da23d3689c327bf11185fa26800ddbdd3). If anyone wants these features as part of the cli, they can request and build from scratch.